projects/plugins/jetpack/extensions/blocks/subscriptions/email-preview.js

```diff
@@ -112,3 +117,155 @@ export default function EmailPreview( { isModalOpen, closeModal } ) {
 		</>
 	);
 }
+
+const devices = [
+	{ name: 'desktop', icon: desktop, label: __( 'Desktop', 'jetpack' ), width: '100%' },
+	{ name: 'tablet', icon: tablet, label: __( 'Tablet', 'jetpack' ), width: '768px' },
+	{ name: 'mobile', icon: mobile, label: __( 'Mobile', 'jetpack' ), width: '360px' },
+];
+
+const DevicePicker = ( { selectedDevice, setSelectedDevice } ) => (
+	<ButtonGroup>
+		{ devices.map( device => (
+			<Button
+				key={ device.name }
+				icon={ device.icon }
+				isSmall
+				isPressed={ selectedDevice === device.name }
+				onClick={ () => setSelectedDevice( device.name ) }
+				style={ {
```

Comment:
Should this be a class? and these values come from the stylesheet?

Reply:
I am trying to use inline on the components that allow for it